plugins/modules/ovirt_host_network.py

```diff
@@ -264,6 +265,8 @@ def get_type_name(mode_number):
             'Load balance (balance-xor)',
             None,
```

Comment:
Just currious why is the Broadcast name None?

Reply:
API does not have any name in it, also for the 6. mode.

Reply:
Hmm, looks like we need to fix HostNicMapper and align it with BondMode enum:

https://github.com/oVirt/ovirt-engine/blob/master/backend/manager/modules/restapi/types/src/main/java/org/ovirt/engine/api/restapi/types/HostNicMapper.java#L34
https://github.com/oVirt/ovirt-engine/blob/50129b39cd9315488e3da37446aee7aa65c0d71e/backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/network/BondMode.java#L45

Reply:
Isn't the `type` attribute of the bond option only informative, and the mode is defined by the `value` attribute? Wouldn't it better to use only the `value` and ignore the `type` in this module?

Reply:
I could try to ignore all options with the name `miimon` not sure if it would work. Just noticed it is also used when creating the network.

Reply:
I do not get the problem, which this code changes tries to solve.

- One possible desired behavior would be to handle the miimon in ansible like any option. This would mean that the user would have to specify the option if he wants it in all modes except 4, and the user should not specify miimon in mode 4, because it will be not reported by oVirt. This would put the work to the user, and would reflect what the user would have to speifcy manually on the host. A problem might be the backward compatibility of the playbooks.
- Another possible desired behavior would be to not report a difference if the miimon is expected to be set, but it is not reported by Engine in mode 4. This would be wrong, if the kernel of the host would remove miimon from defaults, but I guess this will not happen in RHEL8.

I would prefer the first one without the second one, but of course they could be combined.

Reply:
This patch just updates the module so it matches the API. I don't know if the API is correct, that's probably up to you, but I use it as my guidelines for the ansible. 

We don't need to specify the `miimon` from a user. We just get the number of the mode and inside the module, we add to it the required options for the request. This way it does not break backward compatibility.

Reply:
> This patch just updates the module so it matches the API. I don't know if the API is correct, that's probably up to you, but I use it as my guidelines for the ansible.
> 

oVirt -engine REST API does not set the miimon implicitly.

> We don't need to specify the `miimon` from a user. We just get the number of the mode and inside the module, we add to it the required options for the request. This way it does not break backward compatibility.

It might be convinet for the user